README.md

```diff
@@ -113,6 +113,30 @@ See the section: [How does it work?](#how-does-it-work)
 <cidv1> ::= <multibase-prefix><multicodec-cidv1><multicodec-content-type><multihash-content-address>
 ```
 
+### CIDv2 ("Fat Pointers")
+
+CIDv2 offers a way to encode context markers (signalling) into the IPLD link layer.
+
+CIDv2 is used for describing both *data* and the *context* for that data. A specific definition of
+context is not needed, as any context one would wish to signal in the link layer is effectively
+supported.
+
+CIDv2 is, quite literally, two CIDs.
+
+```
+<cidv2> ::= <data-cid><context-cid>
```

Comment:
I hope with `cid` you mean how CIDs are used today and not what the spec says, i.e. these won't contain a multibase prefix.